fastly/block_fastly_service_v1_logging_datadog.go

```diff
@@ -0,0 +1,210 @@
+package fastly
+
+import (
+	"fmt"
+	"log"
+
+	fst "github.com/fastly/go-fastly/fastly"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+var datadogSchema = &schema.Schema{
+	Type:     schema.TypeSet,
+	Optional: true,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			// Required fields
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The unique name of the Datadog logging endpoint",
+			},
+
+			"token": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Sensitive:   true,
+				Description: "The API key from your Datadog account.",
+			},
+
+			// Optional fields
+			"region": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "US",
+				Description: "The region that log data will be sent to. One of `US` or `EU`. Defaults to `US` if undefined.",
+			},
+
+			"format": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Apache-style string or VCL variables to use for log formatting.",
+			},
+
+			"format_version": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      2,
+				Description:  "The version of the custom logging format used for the configured endpoint. Can be either `1` or `2`. (default: `2`)",
```

Comment:
Include "The logging call gets placed by default in vcl_log if format_version is set to 2 and in vcl_deliver if format_version is set to 1."?